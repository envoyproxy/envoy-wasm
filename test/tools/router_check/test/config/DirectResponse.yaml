virtual_hosts:
  - name: www2
    domains:
    - lyft.com
    routes:
      - match:
          path: /ping
        direct_response:
          status: 200
          body:
            inline_string: "Success!"
