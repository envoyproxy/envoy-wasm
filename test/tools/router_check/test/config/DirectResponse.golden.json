[
  {
    "test_name": "DirectResponse",
    "input": {
      ":authority": "lyft.com",
      ":path": "/ping"
    },
    "validate": {
      "path_redirect": "http://lyft.com/ping",
      "cluster_name": "",
      "host_rewrite": "",
      "path_rewrite": "",
      "virtual_cluster_name": "",
      "virtual_host_name": ""
    }
  }
]
